charts/enterprise-kyverno-operator/templates/post-kyverno-upgrade.yaml

```diff
@@ -26,6 +26,13 @@ spec:
             - sh
             - -c
             - |
+              echo "waiting"
+              sleep 60
+              kubectl wait --for=condition=available deployment/kyverno-admission-controller --timeout=60s --namespace=kyverno
```

Comment:
Great. Small suggestion If we could reduce the sleep and increase the timeout to 120s? We might be able to cleanup as soon as we start.

Reply:
I wanted to take care of the condition where the controllers don't terminate soon enough, and the scripts could verify the old ones being available. 

I think I could tweak the condition a bit, let me try